src/test/lib/optimizer/strategy/in_expression_rewrite_rule_test.cpp

```diff
@@ -88,40 +139,7 @@ TEST_F(InExpressionRewriteRuleTest, DisjunctionStrategy) {
     const auto input_lqp = PredicateNode::make(five_element_in_expression, node);
     const auto result_lqp = StrategyBaseTest::apply_rule(rule, input_lqp);
 
-    // Can't use EXPECT_LQP_EQ here, because ExpressionUnorderedSet produces a non-deterministic order of predicates
-    auto values_found_in_predicates = std::vector<int>{};
-
-    // Checks that a given node is a predicate of the form `col_a = x` where x is an int and will be added to
-    // values_found_in_predicates
-    const auto verify_predicate_node = [&](const auto& node) {
-      ASSERT_EQ(node->type, LQPNodeType::Predicate);
```

Comment:
We had to remove the `ASSERT_X` because the code block is moved to a non-void function.